src/scripts/librarymenu.js

```diff
@@ -79,8 +79,8 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
 
     function updateHeaderUserButton(src) {
         if (src) {
-            headerUserButton.classList.add("headerUserButtonRound");
-            headerUserButton.innerHTML = '<img src="' + src + '" />';
+            headerUserButton.classList.add("headerUserButtonRound");            
```

Comment:
Looks like there was some trailing whitespace added here.

Reply:
Still looks like there are some spaces at the end of this line. All other changes look good!